apps/settings/js/panels/search/search.js

```diff
@@ -2,6 +2,7 @@
 
 define(function(require) {
 
+  var Format = require('shared/format'); // jshint ignore:line
```

Comment:
Why the ignore line here? Is it because we don't use the ```Format``` variable?

This is going to scope it to the inner function and not window, so maybe we don't need to assign it to a variable at all?